components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -5999,7 +5985,6 @@ def _getQueryString(cls, opts=None):
                  "left outer join fetch obj.screenLinks spl "
                  "left outer join fetch spl.parent sc")
         # NB: we don't use base _getQueryString.
```

Comment:
No. That comment is still valid. It's just that child_count is not supported anywhere now, so the rest of the comment wasn't relevant now.